### PR TITLE
Feature/form component

### PR DESCRIPTION
**Pull request by martyna-ds**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue) labeled with `bug`
- [x] New feature (non-breaking change which adds functionality) labeled with `enhancement`
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the [code standards](../CONTRIBUTING.md) of this project.
- [ ] My change requires updating the documentation. I have updated the documentation accordingly.

### CODE REVIEW

File: application/backend/src/main/resources/apps/bulma/components/contactform/contactform.html [hunk 1]
```diff
@@ -17,29 +17,28 @@
 */-->
 
 <sly data-sly-use.model="pl.ds.bulma.components.models.ContactFormComponent">
-  <form>
+  <form id="${resource.name}">
```
Comment (paweljozwicki) [inline application/backend/src/main/resources/apps/bulma/components/contactform/contactform.html]: In the case of IDs, much safer would be to use resource.path

**Comment by Rafal-DS** [comment]
I have read the Dynamic Solutions Individual Contributor License Agreement (CLA) Document and I hereby sign the CLA

**Comment by Rafal-DS** [comment]
recheck

File: application/backend/src/main/resources/apps/bulma/components/page/body.html [hunk 2]
```diff
@@ -16,9 +16,11 @@
   ~
 */-->
 
+<sly data-sly-use.model="pl.ds.bulma.components.models.PageComponent">
 <sly data-sly-test.navbarFixed="${properties.navbarFixed}"></sly>
 <sly data-sly-test.position="${properties.navbarFixedPosition}"></sly>
 
-<body class="${properties.defaultTemplateCSS} ${navbarFixed ? position : ''}">
+<body data-config="{'contactForm': '${model.configEndpoint}'}" class="${properties.defaultTemplateCSS} ${navbarFixed ? position : ''}">
```
Comment (paweljozwicki) [inline application/backend/src/main/resources/apps/bulma/components/page/body.html]: I had to change ' with " to make it work:
`data-config='{"contactForm": "${model.configEndpoint}"}'`
I'm not sure if it's only my issue?

File: application/backend/src/main/resources/apps/bulma/components/contactform/contactform.html (hunk 1 above)
Comment (martyna-ds) [reply application/backend/src/main/resources/apps/bulma/components/contactform/contactform.html]: forgot to delete this, I don't use it anyway

File: application/backend/src/main/resources/apps/bulma/components/page/body.html (hunk 2 above)
Comment (martyna-ds) [reply application/backend/src/main/resources/apps/bulma/components/page/body.html]: same with me

Comment (Rafal-DS) [reply application/backend/src/main/resources/apps/bulma/components/page/body.html]: Both versions seem to work for me. Martyna applied Your change already, so its fixed now anyways.

**Review by paweljozwicki — APPROVED** [review]

**Pull request merged**